site_persistence_file.json

```diff
@@ -53,15 +53,9 @@
         },
         {
           "assigner": null, 
-          "system": "http://us.truenth.org/identity-codes/shortcut-alias", 
-          "use": "usual", 
-          "value": "p3pclinica"
-        },
-        {
-          "assigner": null, 
-          "system": "http://us.truenth.org/identity-codes/shortcut-alias", 
-          "use": "usual", 
-          "value": "p3pclinca"
```

Comment:
I assume you realize by removing the shortcut-alias identifiers, any use of them in the /go URL will 404?